pages/item-new/_.vue

```diff
@@ -0,0 +1,369 @@
+        const alternativeTitle = this.altTitle ? langMapValueForLocale(this.altTitle, this.$i18n.locale) : '';
+
+        const allTitles = [].concat(mainTitle, alternativeTitle).filter(Boolean);
+        for (let title of allTitles) {
+          for (let value of title.values) {
+            titles.push({ 'code': title.code, value });
+          }
+        }
+
+        return titles;
+      },
+      descriptionInCurrentLanguage() {
+        if (!this.description) {
+          return false;
+        }
+        return langMapValueForLocale(this.description, this.$i18n.locale);
+      },
+      metaTitle() {
+        return this.titlesInCurrentLanguage[0] ? this.titlesInCurrentLanguage[0].value : this.$t('record.record');
+      },
+      metaDescription() {
+        if (!this.descriptionInCurrentLanguage) return '';
+        return this.descriptionInCurrentLanguage.values[0] ? this.descriptionInCurrentLanguage.values[0] : '';
+      },
+      isRichMedia() {
+        return isRichMedia(this.selectedMedia);
+      },
+      selectedMedia: {
+        get() {
+          return this.selectedMediaItem || this.media[0] || {};
+        },
+        set(about) {
+          this.selectedMediaItem = this.media.find((item) => item.about === about) || {};
+        }
+      },
+      selectedMediaImage() {
+        if (!this.selectedMedia.thumbnails) return {};
+        return {
+          src: this.selectedMedia.thumbnails.large,
+          link: this.isShownAt
+        };
+      },
+      edmRights() {
+        return this.selectedMedia.webResourceEdmRights ? this.selectedMedia.webResourceEdmRights : this.fields.edmRights;
+      },
+      rightsStatement() {
+        if (this.edmRights) return langMapValueForLocale(this.edmRights, this.$i18n.locale).values[0];
+        return false;
+      },
+      dataProvider() {
+        const edmDataProvider = langMapValueForLocale(this.coreFields.edmDataProvider, this.$i18n.locale);
+
+        if (edmDataProvider.values[0].about) {
+          return edmDataProvider.values[0];
+        }
+
+        return edmDataProvider;
+      },
+      notificationUrl() {
+        return `https://classic.europeana.eu/portal/${this.$i18n.locale}/record${this.identifier}.html?utm_source=new-website&utm_medium=button`;
+      },
+      redirectNotificationsEnabled() {
+        return Boolean(Number(process.env.ENABLE_LINKS_TO_CLASSIC));
+      }
+    },
+
+    mounted() {
+      if (process.browser) {
+        this.$gtm.push({
+          itemCountry: langMapValueForLocale(this.fields.edmCountry, 'en').values[0],
+          itemDataProvider: langMapValueForLocale(this.coreFields.edmDataProvider, 'en').values[0],
+          itemProvider: langMapValueForLocale(this.fields.edmProvider, 'en').values[0],
+          itemRights: langMapValueForLocale(this.fields.edmRights, 'en').values[0]
+        });
+      }
+
+      // TODO: remove or is this useful for the new swiper/action?
```

Comment:
Indeed, this'll need to be moved to the itemHero component. It wasn't ready to be picked up on that PR as the IIIF viewer hadn't been added to the swiper then.